### ISSUE TITLE
Bourbon is not detected in Rails 4.2.7

### ISSUE THREAD

**Issue by StarWar** [thoughtbot/bourbon#981]
I was using bootstrap and now I want to use bourbon and neat. So I removed bootstrap related gems and references to it and installed bourbon and neat. It doesn't work
### Bourbon version
  bourbon (~> 5.0.0.beta.7)

### Stack trace
```
File to import not found or unreadable: bourbon.
Load paths:
  /Users/starwars/Sites/DEV_RAILS/my-app-name/app/assets/images
  /Users/starwars/Sites/DEV_RAILS/my-app-name/app/assets/javascripts
  /Users/starwars/Sites/DEV_RAILS/my-app-name/app/assets/stylesheets
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/neat-1.8.0/app/assets/stylesheets
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/redactor-rails-0.5.0/vendor/assets/javascripts
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/redactor-rails-0.5.0/vendor/assets/stylesheets
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/doorkeeper-4.2.0/app/assets/stylesheets
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/doorkeeper-4.2.0/vendor/assets/stylesheets
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/rails_admin-0.8.1/app/assets/images
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/rails_admin-0.8.1/app/assets/javascripts
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/rails_admin-0.8.1/app/assets/stylesheets
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/rails_admin-0.8.1/vendor/assets/javascripts
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/rails_admin-0.8.1/vendor/assets/stylesheets
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/remotipart-1.2.1/vendor/assets/javascripts
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/nested_form-0.3.2/vendor/assets/javascripts
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/jquery-ui-rails-5.0.5/app/assets/images
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/jquery-ui-rails-5.0.5/app/assets/javascripts
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/jquery-ui-rails-5.0.5/app/assets/stylesheets
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/font-awesome-rails-4.5.0.0/app/assets/fonts
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/font-awesome-rails-4.5.0.0/app/assets/stylesheets
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/jquery-rails-4.2.1/vendor/assets/javascripts
  /Users/starwars/.rvm/gems/ruby-2.2.1@rails420/gems/coffee-rails-4.1.1/lib/assets/javascripts
  /Users/starwars/Sites/DEV_RAILS/vendor/assets/stylesheets
```

Extracted source (around line #1):
```
@import "bourbon";
@import "base/base";
```

I downgraded version of bourbon to 4.2.7 and it worked. But then bitters is unable to find some global variables like $font-stack-system etc.

**Comment by tysongach** [comment]
@StarWar If you remove the `rails_admin` gem, does the error go away?

**Comment by tysongach** [comment]
@StarWar Any update on my question above?

**Comment by StarWar** [comment]
@tysongach Sorry for replying late. I did uninstall rails_admin and then updated bourbon to 5.0.0.beta.7 version and it didn't work. It's still not detecting bourbon.

**Comment by NySiya** [comment]
I faced the same problem.

**Comment by tysongach** [comment]
@StarWar @NySiya Can anyone share a code repo we could look at and dig into?

**Comment by tysongach** [comment]
We just published [v5.0.0.beta.8](https://github.com/thoughtbot/bourbon/releases/tag/v5.0.0.beta.8) which potentially solves this issue. Can you try upgrading and let us know if the problem persists or gets resolved?

**Comment by tysongach** [comment]
I’m going to close this because our testing showed that this should be fixed in v5.0.0.beta.8. Feel to re-open if the issue persists.
